src/pages/settings/Payments/PaymentsPage/BasePaymentsPage.js

```diff
@@ -141,6 +140,12 @@ class BasePaymentsPage extends React.Component {
                     type: CONST.PAYMENT_METHODS.PAYPAL,
                 };
             } else if (accountType === CONST.PAYMENT_METHODS.BANK_ACCOUNT) {
+                // 0 means account is pending, send user to add-bank-account to complete setting up personal bank account
+                if (methodID === 0) {
```

Comment:
Can't we use `isPending` instead?

Reply:
Its not being passed in, but I can pass it in as a new param